card/mod/05_standard/lib/carrier_wave/cardmount.rb

```diff
@@ -13,26 +12,33 @@ def uploader_options
       Card.uploader_options ||= {}
     end
 
-    def mount_uploader(column, uploader=nil, options={}, &block)
+    def mount_uploader column, uploader = nil, options = {}, &block
```

Comment:
Method has too many lines. [77/15]<br>Surrounding space detected in default value assignment.